clickhouse/columns/string.cpp

```diff
@@ -232,7 +232,6 @@ void ColumnString::Clear() {
     items_.clear();
     blocks_.clear();
     append_data_.clear();
-    append_data_.shrink_to_fit();
```

Comment:
Why?

Reply:
Do not do shrink_to_fit for reuse the deque  existing memory.  We just erasure the state.